indra/newview/app_settings/settings.xml

```diff
@@ -9153,6 +9153,17 @@
       <real>0.00</real>
     </array>
   </map>
+  <key>RenderTransmissionDetail</key>
```

Comment:
getting a jump on the next extension too eh?

Reply:
Params are all getting wired up tomorrow!